packages/calypso-e2e/src/lib/pages/reader-page.ts

```diff
@@ -0,0 +1,79 @@
+import { Page } from 'playwright';
+
+const selectors = {
+	readerVisitLink: '.reader-visit-link',
+	shareButtonLocator: '.reader-share__button',
+	firstComboCardPostLocator: '.reader-combined-card__post-title-link',
+	siteContentLocator: '.site-selector__sites .site__content',
+	commentButtonLocator: '.comment-button',
+	commentTextAreaLocator: '.comments__form textarea',
+	commentSubmitLocator: '.comments__form button',
```

Comment:
What do you think of making this more specific using a text locator? Like `'.comments__form button:text("Send")'`?

That way, in case another button is added on to the form later, it won't disrupt this!